editor/plugins/editor_resource_tooltip_plugins.cpp

```diff
@@ -107,7 +107,7 @@ Control *EditorTextureTooltipPlugin::make_tooltip_for_path(const String &p_resou
 	vb->set_alignment(BoxContainer::ALIGNMENT_CENTER);
 
 	Vector2 dimensions = p_metadata.get("dimensions", Vector2());
-	Label *label = memnew(Label(vformat(TTR(U"Dimensions: %d × %d"), dimensions.x, dimensions.y)));
+	Label *label = memnew(Label(vformat(TTR(String(U"Dimensions: %d × %d")), dimensions.x, dimensions.y)));
```

Comment:
This would no longer properly extract the translation for weblate, due to the regex that's used to find strings:

https://github.com/godotengine/godot-editor-l10n/blob/main/scripts/extract_editor.py#L52

Reply:
Oh wow, good catch! That will make this case especially awkward. I guess i'd have to patch either the script or add a `char32_t*` constructor to `StringName` just for this....

Reply:
I used `TTRC` in addition to `TTR` for convenience. Let me know if it looks good!